client/embedded-app/src/boards/stm32f767zi.rs

```diff
@@ -8,6 +9,7 @@ pub type NetworkUartTxDma = embassy_stm32::peripherals::DMA1_CH6;
 
 pub struct NetworkPeripherals {
     pub uart: NetworkUart,
+    pub uart_interrupt: USART2,
```

Comment:
Can you please declare a type alias above for the interrupt and use that here?. This faciliates declarations in code.

```rust
pub type NetworkUartInterrupt = interrupt::USART2;
```